tensorflow/contrib/learn/python/learn/learn_io/data_feeder.py

```diff
@@ -433,21 +518,22 @@ def __init__(self, x, y, n_classes, batch_size):
     """Initializes a StreamingDataFeeder instance.
 
     Args:
-      x: iterator that returns for each element, returns features.
-      y: iterator that returns for each element, returns 1 or many classes /
+      x: iterator that returns for each element or dictionary of single element, returns features.
```

Comment:
Can you fix these descriptions to be real sentences? I know you don't broke them, but they're hard to understand.